### PR TITLE
refactor: fix remaining consistency gaps from recent helper extractions

### PR DESCRIPTION
**Pull request by dhruvbatra**
## Summary

Three small cleanups that fell through the cracks across the last few PRs (#88, #90, #96):

- **`formatters.py`**: Route the last hand-rolled `navigator_calls`/`n1_calls` fallback through the `_get_first()` helper — all sibling sites already use it since #88
- **`schemas.py`**: Use `_WEBHOOK_FORMAT_DESCRIPTION` for `EditScoutInput.webhook_format` — the only field not yet using the shared constant hoisted in #90
- **`test_schemas.py`**: Hoist `_output_fields_description` import to module scope (consistent with rest of file) and fix PEP 8 blank line between test classes (from #96)

No behavioral changes. All 141 tests pass.

## Test plan

- [x] Full test suite passes (`pytest tests/ -v` — 141/141)
- [x] `test_activity_falls_back_to_deprecated_n1_calls_key` covers the `_get_first()` change
- [x] `test_all_schemas_use_helper` drift guard still passes after import hoist

cc @dhruvbatra — these are follow-ups to your recent refactoring wave

https://claude.ai/code/session_01DgtVazC9UMC1SHScVSoeg1

---
_Generated by [Claude Code](https://claude.ai/code/session_01DgtVazC9UMC1SHScVSoeg1)_

---

> [!NOTE]
> **Low Risk**
> Low risk refactor-only changes: updates field descriptions and test imports/formatting, plus a comment clarifying a zero-value edge case, with no runtime behavior changes expected.
> 
> **Overview**
> Finishes a few **consistency cleanups** from recent helper extractions.
> 
> Updates `EditScoutInput.webhook_format` to reuse the shared `_WEBHOOK_FORMAT_DESCRIPTION`, and tidies `tests/test_schemas.py` by hoisting the `_output_fields_description` import and fixing minor style/spacing.
> 
> Adds a clarifying comment in `format_usage` explaining why `activity.navigator_calls`/`n1_calls` fallback is *not* routed through `_get_first()` (since `0` is a valid value).
> 
> <sup>Reviewed by [Cursor Bugbot](https://cursor.com/bugbot) for commit b0e91278dacd3642a86010addb3c16acd3992db8. Bugbot is set up for automated code reviews on this repo. Configure [here](https://www.cursor.com/dashboard/bugbot).</sup>

### CODE REVIEW

File: src/yutori_mcp/formatters.py [hunk 1]
```diff
@@ -279,7 +279,7 @@ def format_usage(response: dict[str, Any], **context: Any) -> str:
     if activity:
         period = activity.get("period", "24h")
         # `navigator_calls` is the primary key; `n1_calls` is the deprecated alias.
-        navigator_calls = activity.get("navigator_calls", activity.get("n1_calls", 0))
+        navigator_calls = _get_first(activity, "navigator_calls", "n1_calls", default=0)
```
[the remark this answers is not in the document]
Comment (dhruvbatra) [reply src/yutori_mcp/formatters.py]: Good catch — reverted this line back to the original `.get()` chain in b0e9127. `_get_first()`'s truthiness check would indeed skip a valid `0` count. Added a comment explaining why this call site doesn't use the helper.

---
_Generated by [Claude Code](https://claude.ai/code)_
